seatunnel-api/src/main/java/org/apache/seatunnel/api/source/SeaTunnelSource.java

```diff
@@ -94,6 +94,8 @@ SourceSplitEnumerator<SplitT, StateT> restoreEnumerator(SourceSplitEnumerator.Co
      *
      * @return enumerator state serializer.
      */
-    Serializer<StateT> getEnumeratorStateSerializer();
+    default Serializer<StateT> getEnumeratorStateSerializer(){
+        return new DefaultSerializer<>();
+    }
```

Comment:
```suggestion
    default Serializer<StateT> getEnumeratorStateSerializer() {
        return new DefaultSerializer<>();
    }
```
Keep the code style.

Reply:
done